src/commercetools/store-actions/cart-update-actions.ts

```diff
@@ -0,0 +1,197 @@
+import { Product, TaxCategory } from '@commercetools/platform-sdk';
+import { availablePromotion, ProductToAdd } from '../../integration/types';
+import {
+  OrdersItem,
+  StackableRedeemableResponse,
+  StackableRedeemableResultDiscountUnit,
+} from '@voucherify/sdk';
+import { Cart as CommerceToolsCart } from '@commercetools/platform-sdk/dist/declarations/src/generated/models/cart';
+import getCartActionBuilders from './cart-update-actions/getCartActionBuilders';
+import { checkIfAllInapplicableCouponsArePromotionTier } from '../../integration/utils/helperFunctions';
+import { DataToRunCartActionsBuilder } from './cart-update-actions/CartAction';
+import {
+  CartDiscountApplyMode,
+  PriceSelector,
+  ProductWithCurrentPriceAmount,
+} from '../types';
+import { getCommercetoolstCurrentPriceAmount } from '../utils/getCommercetoolstCurrentPriceAmount';
+import { ByProjectKeyRequestBuilder } from '@commercetools/platform-sdk/dist/declarations/src/generated/client/by-project-key-request-builder';
+
+export class CartUpdateActions implements CartUpdateActions {
```

Comment:
Looks like forgot to import `CartUpdateActions` interface... and now it's a strange construct

```ts
class X implements X {
}
```